km/km_coredump.h

```diff
@@ -111,14 +111,28 @@ typedef struct km_nt_guest {
 #define NT_KM_GUEST 0x4b4d4754       // "KMGT" no null term
 #define NT_KM_DYNLINKER 0x4b4d444c   // "KMDL" no null term
 
+/*
+ * Elf note for dup data
```

Comment:
Could you note that these are only used in snapshots.

Reply:
All of these here are only used in snapshots as described in the commment up above.